File: sdk/client/java/openapi-test/src/test/java/opensdk/sdk/apis/klaytnDebug/blockchainInspection/DebugStartCollectingTrieStatsTest.java
package opensdk.sdk.apis.klaytnDebug.blockchainInspection;

import opensdk.sdk.apis.constant.UrlConstants;
import opensdk.sdk.models.DebugStartCollectingTrieStatsResponse;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
import org.klaytn.OpenSDK;

import java.io.IOException;

@DisplayName("Debug RPC Test")
public class DebugStartCollectingTrieStatsTest {
    private final OpenSDK sdk = new OpenSDK(UrlConstants.SERVER_URL);

    @Test
    @DisplayName("RPC debug_startCollectingTrieStats")
    void whenRequestValid_ThenCall200ResponseReturns() throws IOException {
        String address = "0x0000000000000000000000000000000000000000";

        DebugStartCollectingTrieStatsResponse response = sdk.debug.startCollectingTrieStats(address).send();
        response.getResult();
    }
}

File: sdk/client/javascript/openapi-test/test/debug/blockchainInspection/DebugStartCollectingTrieStats.test.js
const OpenSdk = require("opensdk-javascript");
const { expect } = require("@jest/globals");
const { RPC } = require("../../constant");

const sdk = new OpenSdk(new OpenSdk.ApiClient(RPC));

describe('debug_startCollectingTrieStats API', () => {
    test('should return debug_startCollectingTrieStats', (done) => {

        let callbackOne = function (error, data, response) {
            expect(error).toBeNull();
            expect(data).toBeDefined()
            done();
        };

        const address = "0x0000000000000000000000000000000000000000";

        sdk.debug.startCollectingTrieStats(address, {}, callbackOne);
    });
});

